### PR TITLE
feature cv gerard celma

### PR DESCRIPTION
**Pull request by gcelma**

### CODE REVIEW

File: staff/gerard-celma/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <body>
+        <header>
+            <h1>Waldo Geraldo Faldo</h1>
+            <img src="">
+            <address>street, postcode barcelona, spain</address>
+            <p>Telephone: 666 66 66 66</p>
+            <p>Email: waldo@geraldofaldo.com</p>
+        </header>
+        <main>
+            <section id="about me">
```
Comment (boversauros) [inline staff/gerard-celma/cv/index.html]: You can set space between the words, you should write `about_me` or camelcase `aboutMe`

Comment (boversauros) [reply staff/gerard-celma/cv/index.html]: Sorry I mean can't

File: staff/gerard-celma/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    <li>A self-starter with the ability to work independently and as part of a team.</li>
+                    <li>Ability to network and collaborate cross-functionally.</li>
+                </ul>
+            </section>
+            <section id="education">
+                <h2>Education</h2>
+                <p>Central Birmingham University – Finance Degree 2003 – 2007</p>
+                <p>Coventry School; O levels Maths (A) English (B) Geography (B) Physics (A)</p>
+            </section>
+            <section id="job experience">
```
Comment (boversauros) [inline staff/gerard-celma/cv/index.html]: same as the about me case

File: staff/gerard-celma/cv/index.html (hunk 1 above)
Comment (gcelma) [reply staff/gerard-celma/cv/index.html]: fixed, thanks!

File: staff/gerard-celma/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>CV</title>
+    </head>
+    <body>
+        <header>
+            <h1>Waldo Geraldo Faldo</h1>
+            <img src="">
```
Comment (Arcoders) [inline staff/gerard-celma/cv/index.html]: oups! undefined src and use the alt attribute

File: staff/gerard-celma/cv/index.html (hunk 2 above)
Comment (gcelma) [reply staff/gerard-celma/cv/index.html]: fixed, thanks!

File: staff/gerard-celma/cv/index.html (hunk 3 above)
Comment (gcelma) [reply staff/gerard-celma/cv/index.html]: fixed, thanks!

**Pull request merged**